### PR TITLE
Users/subraman/teskrename

### PR DESCRIPTION
**Pull request by mvvsubbu**
Task Rename and fix for drop down not showing for copyfilestask

### CODE REVIEW

File: Tasks/PowerShellOnTargetMachines/task.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ﻿{
     "id": "3B5693D4-5777-4FEE-862A-BD2B7A374C68",
-    "name": "RunPowerShellOnTargetMachines",
-    "friendlyName": "Run PowerShell on Target Machines",
-    "description": "Run PowerShell on Target Machines.  Executes PowerShell commands remotely on machine(s)",
-    "helpMarkDown": "Run PowerShell on Target Machines.  Executes PowerShell commands remotely on machine(s)",
+    "name": "PowerShellOnTargetMachines",
+    "friendlyName": "PowerShell on Target Machines",
+    "description": "Execute PowerShell scripts on remote machine(s)",
+    "helpMarkDown" : "Execute PowerShell scripts on remote machine(s)",
```
Comment (kmkumaran) [inline Tasks/PowerShellOnTargetMachines/task.json]: Extra space after "helpMarkDown"

File: Tasks/WindowsMachineFileCopy/task.json [hunk 2]
```diff
@@ -63,22 +63,30 @@
             "label": "Clean Target",
             "defaultValue": "false",
             "required": false,
-            "helpMarkDown": "Setting it to true will clean up the destination folder before copying the files."
+            "helpMarkDown": "Setting it to true will clean the destination folder before copying the files."
         },
         {
-            "name": "CopyFilesInParallel",
+            "name": "DeployFilesInParallel",
```
Comment (kmkumaran) [inline Tasks/WindowsMachineFileCopy/task.json]: Retain this as Copy.

**Comment by pryenugu** [comment]
Changes look fine. Make sure to check-in pester test changes, E2E test changes in a single PR at the same time when you commit this change into master.

File: Tasks/WindowsMachineFileCopy/task.json [hunk 3]
```diff
@@ -63,22 +63,30 @@
...
+         "selector": "jsonpath:$.value[*].name", 
+         "authKey": "tfs:DevTestLabs" 
+     } 
+    ], 
+    "instanceNameFormat": "Machine Group: $(EnvironmentName)",
 	 "execution": {
         "PowerShell": {
-            "target": "$(currentDirectory)\\CopyFilesToMachines.ps1",
+            "target": "$(currentDirectory)\\WindowsMachineFileCopy.ps1",
             "argumentFormat": "",
```
Comment (pryenugu) [inline Tasks/WindowsMachineFileCopy/task.json]: Changes look fine. Make sure to check-in pester test changes, E2E test changes in a single PR at the same time when you commit this change into master.

**Pull request closed without merging**